app/v3/controllers/validators/TriggerBsasRulesValidator.scala

```diff
@@ -37,9 +37,9 @@ import javax.inject.{Inject, Singleton}
 class TriggerBsasRulesValidator @Inject() (bsasConfig: BsasConfig) extends RulesValidator[TriggerBsasRequestData] {
 
   private val minYear: Int = 1900
-  private val maxYear: Int = 2100
+  private val maxYear: Int = 2099
 
-  private val resolveDateRange = ResolveDateRange.withLimits(minYear, maxYear)
+  private val resolveDateRange = ResolveDateRange().yearLimited(minYear, maxYear)
```

Comment:
just wondering if the fluent-interface thing can be kept here? Sm like "ResolveDateRange().limitedTo(...)  (or something, if at all)